### PR TITLE
docs(example): converting example to FC

### PR DESCRIPTION
**Pull request by MateusAndrade**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,34 +27,27 @@ npm install --save react-native-view-more-text
 - **textStyle**([object, array]): Styles is passed to `Text` inside `ViewMoreText`
 (Refer to this [PR#8](https://github.com/nlt2390/react-native-view-more-text/pull/8))
 
-```javascript
+```jsx
+  ...redacted
```
Comment (nlt2390) [inline README.md:31]: hi, is this a typo?

Comment (MateusAndrade) [reply README.md:31]: Ah no, i wanted to say that the other part of the code was not needed as it is an example. 😓

Comment (MateusAndrade) [reply README.md]: Done: 488d9c7a2ffcb88aeb76fbb6481f2c49694ff784.

**Comment by nlt2390** [comment]
nice. thanks a lot for your effort to update this outdated readme.

**Pull request merged**